### PR TITLE
Push multiple branches in one call to assign

### LINKED ISSUE #77: Make `--branch` work with multiple branches
So you could do something like `assign hw1 --branch master,devel`, rather than running `assign` twice.

### PR DESCRIPTION
**Pull request by LinuxMercedes**
Closes #77 (and fixes part of #59).

### CODE REVIEW

File: assigner/baserepo.py [hunk 1]
```diff
@@ -176,6 +176,10 @@ def clone_to(self, dir_name, branch=None):
             if branch:
                 self._repo = git.Repo.clone_from(self.ssh_url, dir_name,
                                                 branch=branch)
+                for b in branch:
+                    self._repo.create_head(b, 'origin/{}'.format(b))
```
Comment (brhoades) [inline assigner/baserepo.py]: np: we should stick to double quotes, I think that's what the rest of the file is using.

**Comment by brhoades** [comment]
This looks good, let's just make it consistent.

File: assigner/commands/assign.py [hunk 2]
```diff
@@ -64,7 +64,8 @@ def assign(conf, args):
                     repo = StudentRepo.new(base, semester, student_section,
                                            username, token)
                     repo.push(base, branch)
```
Comment (brhoades) [inline assigner/commands/assign.py]: and repo.push is ok with a list?

Comment (LinuxMercedes) [reply assigner/commands/assign.py]: Yep, it will push multiple branches if you ask it to!

**Comment by LinuxMercedes** [comment]
Since commas are valid in git branch names, I switched from the hacky way of splitting up the command line list  to using argparse the way it wants to be used. (Spaces aren't valid in git branch names, so space-delimiting works great.)

Also updated the tutorial to explain how to use this feature.

**Pull request merged**